src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Mapping/ToViewModel/AllOtherNotesViewModelMapTests.cs

```diff
@@ -0,0 +1,140 @@
+
+            //assert
+            exception.Should().BeOfType<ArgumentNullException>();
+        }
+
+        [Fact]
+        public void Map_GivenListOfEvidenceNotes_MapperShouldBeCalled()
+        {
+            //arrange
+            var notes = new List<EvidenceNoteData>
+            {
+                 fixture.Create<EvidenceNoteData>(),
+                 fixture.Create<EvidenceNoteData>(),
+                 fixture.Create<EvidenceNoteData>()
+            };
+
+            var organisationId = Guid.NewGuid();
+            var aatfId = Guid.NewGuid();
+ 
+            var transfer = new EvidenceNotesViewModelTransfer(organisationId, aatfId, notes);
+
+            //act
+            map.Map(transfer);
+
+            // assert 
+            A.CallTo(() => mapper.Map<List<EvidenceNoteRowViewModel>>(notes)).MustHaveHappenedOnceExactly();
+        }
+
+        [Fact]
+        public void Map_GivenAnEmptyListOfEvidenceNotes_MapperShouldNotBeCalled()
+        {
+            //arrange
+            var notes = new List<EvidenceNoteData>();
+
+            var organisationId = Guid.NewGuid();
+            var aatfId = Guid.NewGuid();
+
+            var transfer = new EvidenceNotesViewModelTransfer(organisationId, aatfId, notes);
+
+            //act
+            map.Map(transfer);
+
+            // assert 
+            A.CallTo(() => mapper.Map<EvidenceNoteRowViewModel>(A<EvidenceNoteRowViewModel>._)).MustHaveHappened(0, Times.Exactly);
+        }
+
+        [Fact]
+        public void Map_GivenAnEmptyListOfEvidenceNotes_MustReturnAnEmptyModel()
+        {
+            //arrange
+            var notes = new List<EvidenceNoteData>();
+
+            var organisationId = Guid.NewGuid();
+            var aatfId = Guid.NewGuid();
+
+            var transfer = new EvidenceNotesViewModelTransfer(organisationId, aatfId, notes);
+
+            //act
+            var result = map.Map(transfer);
+
+            // assert 
+            result.EvidenceNotesDataList.Should().BeNullOrEmpty();
+        }
+
+        [Fact]
+        public void Map_GivenListOfEvidenceNoteData_ShouldReturnMappedData()
+        {
+            //arrange
+            var notes = fixture.CreateMany<EvidenceNoteData>().ToList();
+
+            var returnedNotes = new List<EvidenceNoteRowViewModel>
+            {
+                 fixture.Create<EvidenceNoteRowViewModel>(),
+                 fixture.Create<EvidenceNoteRowViewModel>(),
+                 fixture.Create<EvidenceNoteRowViewModel>()
+            };
+
+            var model = new EditDraftReturnedNotesViewModel
+            {
+                EvidenceNotesDataList = returnedNotes
+            };
+
+            var organisationId = Guid.NewGuid();
+            var aatfId = Guid.NewGuid();
+
+            var transfer = new EvidenceNotesViewModelTransfer(organisationId, aatfId, notes);
+            A.CallTo(() => mapper.Map<List<EvidenceNoteRowViewModel>>(notes)).Returns(returnedNotes);
```

Comment:
would you add it to the assert area - A.CallTo(() => mapper.Map<List<EvidenceNoteRowViewModel>>(notes)).MustHaveHappenedExactylOnce()